DependencyInjection/Configuration.php

```diff
@@ -425,6 +425,9 @@ public function getConfigTreeBuilder()
                                     ->thenInvalid('PHP >= 5.4.0 is required to use traits.')
                                 ->end()
                             ->end()
+                            ->integerNode('priority')
+                                ->defaultValue(0)
```

Comment:
Please document this with a call to `info()`, for instance does 0 mean high or low priority?